### ISSUE TITLE
Two very quick questions about training

### ISSUE THREAD

**Issue by greeneggsandyaml** [cvlab-columbia/zero123#54]
Hello, thank you for your nice work! I have two quick questions about training. 

(1) When do you see the network's ability to control viewpoint emerge? I have trained models for around 10,000 steps, but they seem completely unable to control viewpoint. Is this normal? 

(2) How do you store/load data while training? I'm finding that with a batch size of 192 per GPU across 8 GPUs, dataloading becomes a bottleneck. Is this something you also encountered? 

Thank you so much for your help!

**Comment by ruoshiliu** [comment]
Hi @greeneggsandyaml , thank you for your interesting in our work and your questions.
1. Ability to control viewpoint should definitely emerge before 10k steps, assuming you are using the same hyperparameters as we did. The accuracy and quality will improve when you train longer.
2. We store / load data from a local nvme drive installed in the GPU machine. HDD drives should also be enough and won't become a bottleneck. If you are using NFS or other storage systems with high latency for random memory access, I would suggest you preprocess the dataset or use [webdataset](https://github.com/webdataset/webdataset).

Hope this helps.
